app/home/home.tsx

```diff
@@ -35,11 +38,15 @@ export default function Home() {
           <div className="text-center text-md-start ">
             <h1 className="img-overlay">Dołącz do grona sztabowiczów!</h1>
             <p className="px-5 px-md-0 img-overlay">
-              Kończymy urlopy i&nbsp;zabieramy
-              się&nbsp;do&nbsp;pracy. Każda&nbsp;para&nbsp;rąk&nbsp;się&nbsp;przyda!
+              Kończymy urlopy i&nbsp;zabieramy się&nbsp;do&nbsp;pracy.
+              Każda&nbsp;para&nbsp;rąk&nbsp;się&nbsp;przyda!
             </p>
             <h6 className="img-overlay">Kliknij po więcej informacji!</h6>
-            <Button variant="primary" size="lg" href={collaborationNavigation.link}>
+            <Button
+              variant="primary"
+              size="lg"
+              href={collaborationNavigation.link}
```

Comment:
href w Button zamienia go na tag <a>, pomysl aby zamiast tego dać cos co podmieni ścieżkę, a nie przekieruje
https://nextjs.org/docs/pages/building-your-application/routing/linking-and-navigating#imperative-routing